lib/command/stats.js

```diff
@@ -15,7 +15,8 @@ module.exports = async function statsCommand(commandOptions) {
 
 	const channelStats = uniqueChannelIds.map(uniqueChannelId => {
 		const count = channelIds.filter(channelId => channelId === uniqueChannelId).length;
-		return `*<#${uniqueChannelId}>*: ${numberFormatter.format(count)} races`;
+		const noun = (count === 1) ? 'race' : 'races';
+		return `*<#${uniqueChannelId}>*: ${numberFormatter.format(count)} ${noun}}`;
```

Comment:
Think there's an extra closing curly?
```suggestion
		return `*<#${uniqueChannelId}>*: ${numberFormatter.format(count)} ${noun}`;
```

Reply:
Oof sorry! 🤦
I tested it in my browser console and then pasted into GitHub with all the diligence of a hungry bear

Reply:
Fixed